src/redux/actions/concepts/dictionaryConcepts.js

```diff
@@ -144,6 +144,31 @@ export const fetchDictionaryConcepts = (
   dispatch(isFetching(false));
 };
 
+export const fetchConceptsByName = query => async (dispatch) => {
+  dispatch(isFetching(true));
+  const conceptType = localStorage.getItem('type');
+  const conceptOwner = localStorage.getItem('typeName');
```

Comment:
How is concept owner and type name related?

Reply:
@dkayiwa I have changed the constant to `userTypeName `